server.py

```diff
@@ -0,0 +1,126 @@
+#python -m uvicorn server:app --reload
+
+import sqlite3
+import argparse  
+import random
+import string
+import sqlite_helpers
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
+from datetime import datetime, timedelta
+
+app = FastAPI()
+
+#maybe create the table if it doesnt already exist
+sqlite_helpers.maybe_create_table("urldatabase.db")
+
+@app.post("/create_url")
+async def create_url(request: Request):
+
+    urljson = await request.json()
+    timestamp = datetime.now()
+
+    if "url" not in urljson:
+        raise HTTPException(status_code=400, detail="URL required.")
+
+    if "alias" not in urljson:
+        aliasVal = generate_alias()
+    else:
+        aliasVal = urljson['alias']
+
+    if sqlite_helpers.insert_url("urldatabase.db", urljson['url'], aliasVal):
+        url_data = {"url": urljson['url'], "alias": aliasVal, "timestamp": timestamp, "message": "URL added successfully"}
+        return url_data
+    else:
+        raise HTTPException(status_code=409, detail="alias taken nerd")
+    
+    
+
+@app.get("/get_urls")
+async def get_all_urls():
+    return sqlite_helpers.get_urls("urldatabase.db")
+
+
+@app.get("/get_url/{alias}")
+async def get_url(alias: str):
+
+    url_output = sqlite_helpers.get_url("urldatabase.db", alias)
+    if url_output is None:
+        raise HTTPException(status_code=404, detail="URL not found.")
+    
+    return RedirectResponse(url_output)
+    
+
+@app.post("/delete_url/{alias}")
+async def delete_url(alias: str):
+    if(sqlite_helpers.delete_url("urldatabase.db", alias)):
+        return {"message": "URL deleted successfully"}
+    else:
+        raise HTTPException(status_code=404, detail="URL not found.")
+
+@app.exception_handler(HTTPException)
+async def http_exception_handler(request, exc):
+    if exc.status_code == 400:
```

Comment:
address all the other comments and then when you get to this one, lmk and ill teach you python enums